### PR TITLE
Create more apis

### PR DESCRIPTION
**Pull request by carolSequeira**
Added new apis 
refactored existing code
made use of context to get user info

@samjeyam @vijayenthran @rwchan please check

### CODE REVIEW

File: fancy-team-server/Services/Event/index.ts [hunk 1]
```diff
@@ -1,59 +1,105 @@
...
 
+export class EventService {
 
+    public static findEvent = async (id: number) => {
+        try {
+            const eventRepository = getConnection().getRepository(Event);
+            return await eventRepository.findOne({id: id});
+        }
+        catch (error) {
+            console.log(error);
```
Comment (samjeyam) [inline fancy-team-server/Services/Event/index.ts:23]: you need to throw the error and not loose it

File: fancy-team-server/Services/Event/index.ts [hunk 2]
```diff
@@ -1,59 +1,105 @@
...
-    return eventRepository.save(event).then((result) => {
-        return result;
-    });
-};
+    public static updateEvent = async (id: number, title: string, eventType: string, eventDate: string,
+                                       startTime: string, endTime: string, locationName: string,
+                                       description: string, deadline: string, emails: string[]) => {
+       try {
+           await UserEventStatusService.addInvitees(id, emails);
+           return await getConnection()
```
Comment (samjeyam) [inline fancy-team-server/Services/Event/index.ts]: Isn't the this `return` redundant

File: fancy-team-server/Services/Event/index.ts [hunk 3]
```diff
@@ -1,59 +1,105 @@
...
+          event.description = description;
+          event.deadlineDate = deadline;
+
+
+          await emailService.send('sam.thambu@sap.com', emails, {
+              event_link: 'www.gooogle.com',
+              event_name: 'hello world'
+          });
+
+          return eventRepository.save(event).then(async (result) => {
```
Comment (samjeyam) [inline fancy-team-server/Services/Event/index.ts]: why mix promise and async/await here ?

**Review by samjeyam — CHANGES_REQUESTED** [review]

File: fancy-team-server/Services/Event/index.ts (hunk 2 above)
Comment (carolSequeira) [reply fancy-team-server/Services/Event/index.ts]: why is this redundant I have only one return here

**Pull request merged**